internal/signer/linux/util/test_data/certificate_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cert_configs": {
+    "pkcs11": {
+      "slot": "0x1739427",
+      "label": "gecc",
+      "module_path": "pkcs11_module.so"
```

Comment:
nit: field name "module" to be consistent, since we don't have _path suffix elsewhere in the libs config section